content/docs/learning-guide/tutorials/ai/kythera-city-level-overview.md

```diff
@@ -0,0 +1,219 @@
+---
+linkTitle: Kythera City Level Overview
+title: Kythera City Level Overview
+description: Overview of the AI demo level "Kythera City"
+weight: 100
+---
+
+![A ship navigates Kythera City in 3D](/images/learning-guide/tutorials/ai/kythera-city-ship.png)
+
+The `KytheraCity` level demonstrates simple examples of setup for several Kythera AI features, in the Open3D (developers preview) build of the Kythera AI gem.
+
+The Kythera AI gem features demonstrated in this level are:
+
+*   [2D Navmesh generation](/docs/user-guide/gems/reference/kythera-ai/navmesh-generation) and customizability
```

Comment:
```suggestion
*   [2D navmesh generation](/docs/user-guide/gems/reference/kythera-ai/navmesh-generation) and customizability
```
For capitalization consistency with most occurrences of "navmesh" in the linked article, I recommend making "Navmesh" lower case here, too.